bin/cli.js

```diff
@@ -491,8 +487,12 @@ For more information, see https://webpack.js.org/api/cli/.`);
 					process.stdin.resume();
 				}
 				compiler.watch(watchOptions, compilerCallback);
-				if (outputOptions.infoVerbosity !== "none") console.log("\nwebpack is watching the files…\n");
-			} else compiler.run(compilerCallback);
+				if (outputOptions.infoVerbosity !== "none") console.error("\nwebpack is watching the files…\n");
+				if (compiler.close) compiler.close(compilerCallback);
```

Comment:
Why not add 491 and 494 outside of the if statement into its own branch? (be sibling of `if(firstOptions.watch...`)